_includes/dochead.html

```diff
@@ -34,7 +34,7 @@
     var trackOutboundLink=function(n){ga("send","event","outbound","click",n,{hitCallback:function(){document.location=n}})};
   </script>
  </head>
- <body>
+ <body class="article">
```

Comment:
Ok, i thought that might be the case. My last suggestion would be rename this class so that its clear what its purpose is? apioverride?